hokusai/commands/stack.py

```diff
@@ -1,72 +1,72 @@
 import os
+import yaml
 
-from subprocess import check_output, check_call, CalledProcessError, STDOUT
+from hokusai.command import command
+from hokusai.config import config
+from hokusai.common import print_red, print_green, shout
+from hokusai.deployment import Deployment
+from hokusai.service import Service
+from hokusai.kubectl import Kubectl
 
-from hokusai.config import HokusaiConfig
-from hokusai.common import print_red, print_green, verbose, select_context, HokusaiCommandError
-
-def stack_up(context):
-  HokusaiConfig().check()
+@command
+def stack_create(context):
   kubernetes_yml = os.path.join(os.getcwd(), "hokusai/%s.yml" % context)
   if not os.path.isfile(kubernetes_yml):
     print_red("Yaml file %s does not exist for given context." % kubernetes_yml)
     return -1
 
-  try:
-    select_context(context)
-  except HokusaiCommandError, e:
-    print_red(repr(e))
-    return -1
+  kctl = Kubectl(context)
+  shout(kctl.command("create -f %s" % kubernetes_yml))
+  print_green("Stack %s created" % context)
 
-  try:
-    check_call(verbose("kubectl apply -f %s" % kubernetes_yml), shell=True)
-  except CalledProcessError, e:
-    print_red("Stack up failed with error: %s" % e.output)
+@command
+def stack_update(context):
+  kubernetes_yml = os.path.join(os.getcwd(), "hokusai/%s.yml" % context)
+  if not os.path.isfile(kubernetes_yml):
+    print_red("Yaml file %s does not exist for given context." % kubernetes_yml)
     return -1
 
-  print_green("Stack %s created" % kubernetes_yml)
-  return 0
+  kctl = Kubectl(context)
+  shout(kctl.command("apply -f %s" % kubernetes_yml))
+  print_green("Stack %s updated" % context)
 
-def stack_down(context):
-  HokusaiConfig().check()
+@command
+def stack_destroy(context):
```

Comment:
this won't destroy you local config file for the given context - is that clear to the end user?